Rubberduck.Core/AddRemoveReferences/ReferenceReconciler.cs

```diff
@@ -67,25 +75,38 @@ public List<ReferenceModel> ReconcileReferences(IAddRemoveReferencesModel model,
                         reference.Dispose();                        
                     }
                 }
-
                 output.AddRange(selected.Values.OrderBy(selection => selection.Priority)
                     .Select(reference => TryAddReference(project, reference)).Where(added => added != null));
             }
 
+            UpdateSettings(model, true);
             return output;
         }
 
+        private static readonly List<string> InterestingExtensions = new List<string> { ".olb", ".tlb", ".dll", ".ocx", ".exe" };
```

Comment:
How will we handle the rare situation where the type library is in a nonstandard file extension?

Reply:
If it's still referenced after Apply or OK is clicked, it will still get added. The only difference should be that it will initially show as broken in the dialog.